Rakefile

```
@@ -72,7 +72,11 @@ namespace :dev do
   namespace :version do
     desc "Bump version for new development"
     task :bump do
-      File.write("base_version", env_var("NEW_VERSION", version.succ))
+      next_version = version.succ
```

こうじゃないと`NEW_VERSION`を指定したときに壊れない？

```suggestion
      next_version = env_var("NEW_VERSION", version.succ)
```

---

たしかに…。